website/src/edit-word-panel.tsx

```diff
@@ -0,0 +1,127 @@
+import React from "react"
+import { GoPencil } from "react-icons/go"
+import { IoCheckmark } from "react-icons/io5"
+import { Button } from "reakit/Button"
+import {
+  unstable_Form as Form,
+  unstable_FormInput as FormInput,
+  unstable_FormLabel as FormLabel,
+} from "reakit/Form"
+import { unstable_FormSubmitButton as FormSubmitButton } from "reakit/Form"
+import {
+  buttonStyle,
+  cleanButtonStyle,
+  formInputLabelStyle,
+  formStyle,
+  inputStyle,
+  textareaStyle,
+} from "./edit-word-panel.css"
+import { useForm } from "./form-context"
+
+/** Button that allows user to enter edit mode in the word panel, and edit fields of a word. */
+export const EditButton = () => {
+  const { form, isEditing, setIsEditing } = useForm()
+
+  return (
+    <Form {...form} className={formStyle}>
+      {isEditing ? (
+        // Displays a "Cancel" button and "Save" button in editing mode.
+        <>
+          <Button
+            className={cleanButtonStyle}
+            aria-label="Cancel editing the selected word and its features"
+            onClick={() => {
+              setIsEditing(false)
+            }}
+          >
+            Cancel
+          </Button>
+
+          <FormSubmitButton
+            {...form}
+            className={buttonStyle}
```

Comment:
For reference if you can replace this with a preset button component, it should let you do `<XButton as={FormSubmitButton}`. It seems like our library is missing a component for a button with icon _and_ text, perhaps. If so then extract out your styles into a reusable component.

Reply:
Cool, I went ahead and extracted it into a reusable component!